data/planet/gallium/ocaml-compiler-weekly-3-april-2023.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: OCaml compiler weekly, 3 April 2023
 description:
-url: http://gallium.inria.fr/blog/florian-compiler-weekly-2023-04-03
+url: http://gallium.inria.fr/blog/florian-compiler-weekly
```

Comment:
Rather `http://gallium.inria.fr/blog/florian-weekly-2023-04-03` ?

Reply:
ahh, thank you!